config/default/common/config/wv.json/layers/modis/terra/MODIS_Terra_Thermal_Anomalies_Day.json

```diff
@@ -15,7 +15,7 @@
       "product": "MOD14",
       "type": "vector",
       "transition": "true",
-      "startDate": "2012-05-08",
+      "wrapadjacentdays": false,
```

Comment:
Do we need to define this property with a `false` value or can we just leave it out entirely?

Reply:
not sure! it was in there as 'false' for some layers, so i just left it at that. We can delete it if it makes more sense

Reply:
Let's go ahead and remove it from all layers where it's defined as false.  I'm going through all of our configs/properties while preparing for the vis metadata work and trying to remove superfluous config properties like this.  

If the behavior is the same without it, I think it's cleaner to leave it out entirely 👍

Reply:
great - removed!